### DISCUSSION TITLE
How to set automated rollback on argocd after 10 min if pods are not up successfully!

### DISCUSSION THREAD

**Discussion by Tanmoy037** [argoproj/argo-rollouts#3937] [Q&A]
I have more than 30 microservices. I want to rollback pods in non-prod env, If pods are not up successfully after 10 min.

**Comment by kostis-codefresh** [comment]
Check the progressDeadlineSeconds and progressDeadlineAbort properties of the Rollout Spec

https://argo-rollouts.readthedocs.io/en/stable/features/specification/
